src/health-score.js

```diff
@@ -11,10 +39,16 @@ module.exports = {
    */
   compile: async function compileScore(core, github) {
     // TODO: wire up action outputs
+    const extensionInput = core.getInput('extension');
+    const includeInput = core.getInput('include');
+    const excludeInput = core.getInput('exclude');
+
+    const extensions = parseYamlArray(extensionInput);
+    const includes = parseYamlArray(includeInput);
+    const excludes = parseYamlArray(excludeInput);
     return {
-      // TODO: support src as arrays
       // TODO: wire up exclude input
```

Comment:
I think this comment can now be gone since you wired it up?

Reply:
Removed.